packages/flutter_tools/lib/src/web/chrome.dart

```diff
@@ -162,13 +162,13 @@ class ChromiumLauncher {
 
   /// Launch a Chromium browser to a particular `host` page.
   ///
-  /// `headless` defaults to false, and controls whether we open a headless or
-  /// a `headfull` browser.
+  /// [headless] defaults to false, and controls whether we open a headless or
+  /// a "headfull" browser.
```

Comment:
[Headful](https://www.merriam-webster.com/dictionary/headful#:~:text=%3A%20a%20quantity%20(as%20of%20information)%20that%20fills%20the%20head)? I have no idea :)

Reply:
not headless .. heady? 🤷‍♂️ I did write this after all